src/main/java/roboy/ros/RosMainNodeOffline.java

```diff
@@ -0,0 +1,66 @@
+package roboy.ros;
```

Comment:
rather check in the state is we're in the offline mode and don't use ROS in that case
so we get rid of this class